### ISSUE TITLE
NPE on server start-up

### ISSUE THREAD

**Issue by SlimeDog** [pop4959/ChunkyBorder#1]
Spigot 1.16.3
Chunky 1.1.14
ChunkyBorder 1.0.0

On server start-up, the log reports
```
[15:35:27] [Server thread/INFO]: [ChunkyBorder] Enabling ChunkyBorder v1.0.0
[15:35:27] [Server thread/ERROR]: Error occurred while enabling ChunkyBorder v1.0.0 (Is it up to date?)
java.lang.NullPointerException: null
        at org.popcraft.chunkyborder.ChunkyBorder.onEnable(ChunkyBorder.java:107) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugin(CraftServer.java:494) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugins(CraftServer.java:408) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.MinecraftServer.loadWorld(MinecraftServer.java:435) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.DedicatedServer.init(DedicatedServer.java:216) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:808) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$0(MinecraftServer.java:164) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```
and later, before any players connect
```
[15:36:38] [Server thread/ERROR]: Could not pass event PlayerTeleportEvent to ChunkyBorder v1.0.0
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:319) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.craftbukkit.v1_16_R2.entity.CraftPlayer.teleport(CraftPlayer.java:640) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.craftbukkit.v1_16_R2.entity.CraftEntity.teleport(CraftEntity.java:468) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.citizensnpcs.npc.CitizensNPC.spawn(CitizensNPC.java:272) ~[?:?]
        at net.citizensnpcs.npc.CitizensNPC.load(CitizensNPC.java:162) ~[?:?]
        at net.citizensnpcs.api.npc.SimpleNPCDataStore.loadInto(SimpleNPCDataStore.java:59) ~[?:?]
        at net.citizensnpcs.Citizens$3.run(Citizens.java:339) ~[?:?]
        at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftTask.run(CraftTask.java:81) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:400) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.MinecraftServer.b(MinecraftServer.java:1059) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.DedicatedServer.b(DedicatedServer.java:349) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.MinecraftServer.a(MinecraftServer.java:1007) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:846) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$0(MinecraftServer.java:164) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
        at org.popcraft.chunkyborder.ChunkyBorder.onPlayerTeleport(ChunkyBorder.java:208) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:315) ~[spigot-1.16.3.jar-2020-09-26-2133:git-Spigot-17d78db-a339310]
        ... 17 more
```
several times.

**Comment by pop4959** [comment]
Someone already reported this on Discord, we think it's fixed. Can you try downloading the latest version on Spigot?

**Comment by SlimeDog** [comment]
Confirmed fixed with ChunkyBorder-1.0.1

PS: Discord is not a bug tracking system. :)

**Comment by pop4959** [comment]
@SlimeDog I'm aware; there is no way that you would have known that. Thanks for reporting here too! Glad to hear it's fixed on your end as well... another person and myself were not able to reproduce the issue ourselves, but it was fairly evident where the error was coming from, so it's fixed now. Funny day zero issues.
